scss/components/_selectItemsDialog.scss

```diff
@@ -73,6 +73,19 @@
     .dialog-button-box button {
         -moz-window-dragging: no-drag;
     }
+
+	// Move padding into rows so that draggable collection container
+	// does not expand all the way past the scrollbar and cause draggin scrollbar
+	// to drag the entire window.
+	#zotero-collections-tree-container {
+		.virtualized-table-body {
+			padding-inline-end: 0;
+			.row {
+				padding-inline-end: 8px;
+				width: 183px;
```

Comment:
Why 183px?

Reply:
After moving padding around, I noticed that the collection rows would stretch all the way to the end, so to leave some space between them and the scrollbar, I set the width that the rows initially had, prior to this css tweak, which was 183px.

<img width="388" alt="Screenshot 2024-10-10 at 1 56 35 PM" src="https://github.com/user-attachments/assets/8e218bba-ae71-455f-b08b-1d9ba4686c46">

I realize it is a somewhat hacky. My intention was to still have the edges and bottom of the left sidebar allow window drag. It is quite possible that it's not worth the hack, in which case we can just make the entire left pane `-moz-window-drag: no-drag`.